thunder/tests/test_randomness.py

```diff
@@ -20,8 +20,63 @@ def test_uniform_philox(executor, device: str, dtype: dtypes.dtype):
     def func(shape, dtype, device, rng_seed, rng_offset):
         return ltorch.uniform_philox(shape, device=device, dtype=dtype, seed=rng_seed, offset=rng_offset)
 
-    cf = lc_compile(func, disable_preprocessing=True, executors_list=executor.executors_list())
+    cf = thunder.jit(func, executors_list=executor.executors_list())
```

Comment:
Yay!

Reply:
IMHO this should be done in a separate pr, ideally

Reply:
Yan, could you please create a separate PR with this and similar changes that we would merge before the current PR?

Reply:
Done in https://github.com/Lightning-AI/lightning-thunder/pull/377.